cmd/create.sh

```diff
@@ -5,5 +5,9 @@ cmd_create_help() {
 
 _EOF
 }
+rename_function cmd_create ds_cmd_create
 
-
+cmd_create(){
+	#there are no log file for jekyll in offical version so no log file mount	
+	ds_cmd_create
+}
```

Comment:
You can mount the directory of the projects.
If there are any configuration files you can mount them too, or their directory.